test/schemes/fluid/viscosity.jl

```diff
@@ -0,0 +1,49 @@
+include("../../test_util.jl")
+
+@testset verbose=true "Viscosity" begin
+    particle_spacing = 0.2
+    smoothing_length = 1.2 * 0.5 * particle_spacing
+    smoothing_kernel = SchoenbergCubicSplineKernel{2}()
+    sound_speed = 10 * sqrt(9.81 * 0.9)
+
+    state_equation = StateEquationCole(; sound_speed, reference_density=1000.0,
+                                       exponent=7, clip_negative_pressure=false)
+
+    fluid =  rectangular_patch(particle_spacing, (3, 3), seed=1)
+
+    system_wcsph = WeaklyCompressibleSPHSystem(fluid, ContinuityDensity(),
+    state_equation, smoothing_kernel, smoothing_length)
```

Comment:
```suggestion
```
Can't you omit all of this and just hardcode a reasonable kernel gradient as a vector similar to `v_diff` and `pos_diff`?